scripts/errcheck.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Check gofmt
+echo "==> Checking AWS provider for unchecked errors..."
+echo "==> NOTE: at this time we only look for uncheck errors in the AWS package"
+
+if ! which errcheck > /dev/null; then
+    echo "==> Installing errcheck..."
+    go get -u github.com/kisielk/errcheck
+fi
+
+err_files=$(errcheck -ignoretests -ignore \
+  'github.com/hashicorp/terraform/helper/schema:Set' \
+  -ignore 'bytes:.*' \
+  -ignore 'io:[cC]lose|[wWrite]' \
```

Comment:
Do we need to match the lowercase `c` or `w` for anything? Also you have brackets around the entire `[wWrite]`, matching any single one of those characters.